service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.events.SubmittedCaseEvent;
+import uk.gov.hmcts.reform.fpl.service.HmctsCourtLookUpService;
+import uk.gov.hmcts.reform.fpl.service.LocalAuthorityService;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@Component
+public class NotificationHandler {
+
+    private final HmctsCourtLookUpService hmctsCourtLookUpService;
+    private final LocalAuthorityService localAuthorityService;
+    private final NotificationClient notificationClient;
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private final static String JURISDICTION = "PUBLICLAW";
+    private final static String CASE_TYPE = "Shared_Storage_DRAFTType";
+
+    @Value("${ccd.ui.base.url}")
+    String CCD_BASE_URL;
```

Comment:
Let's declare `private final String uiBaseUrl` and lets inject `${ccd.ui.base.url}` via constructor.